<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>441</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Plant List</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>511</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Active Plants</string>
      </attribute>
      <widget class="QTableView" name="plantTableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>481</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Archived Plants</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="plantDetailBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>451</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Selected Plant</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>431</width>
       <height>56</height>
      </rect>
     </property>
     <property name="title">
      <string>Plant Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="submitEditBtn">
        <property name="text">
         <string>Submit Edits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Archive Plant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="markWateredBtn">
        <property name="text">
         <string>Mark as Watered</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetEditBtn">
        <property name="text">
         <string>Reset Edits</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="plantTypeBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Attributes</string>
     </property>
     <widget class="QCheckBox" name="flowering">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Flowering</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tropical">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Tropical</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="succulent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Succulent</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cactus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cactus</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="foliage">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Foliage</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fern">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Fern</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="edible">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Edible</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="aquatic">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Aquatic</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="petsafe">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Pet Safe</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="airplant">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Air Plant</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tree">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Tree</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="conifer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Conifer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Notes</string>
     </property>
     <widget class="QTextEdit" name="notesEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Basics</string>
     </property>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="plantName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="dateGotLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Acquired</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateObtainEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>500</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>status label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>191</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Water/Fertilizer</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Watered</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="lastWateredDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Scheduled Watering</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="nextWaterDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>169</width>
        <height>40</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Water every</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="waterDays">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>days!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Needs fertilizer?</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QLabel" name="needsWaterLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>4</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>4</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>I need to be watered!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>120</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QPushButton" name="addPlantBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Plant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Garden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Garden</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#e9e9e9&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:24pt; color:#005500;&quot;&gt;VERDANT VENTURE&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; color:#005500;&quot;&gt;garden management system&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="notificationBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>120</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Notifications</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
